pyproject.toml

```diff
@@ -78,6 +78,8 @@ django-taggit = "~1.5.1"
 # Represent time zones in Django
 # NOTE: django-timezone-field 4.2.x is available but appears to break our initial migrations?
 django-timezone-field = "~4.1.2"
+# Tree database structures based on Common Table Expressions
+django-tree-queries = "~0.11"
```

Comment:
Are we OK with shipping 1.4 with it depending on *both* django-mptt and django-tree-queries?

Reply:
IMHO: Yes. For now we have seen a bit of roadblocks with both (maintained vs feature compute) and we aren't providing/pushing those interfaces directly to users.

We still need to evaluate if we can leave `django-mptt` comfortably which I don't feel we need rush at the moment.

Reply:
No, but is it possible to rip out MPTT tho and still be backwards-compatible?